test/factories/members.rb

```diff
@@ -15,5 +15,9 @@
     password_confirmation { password }
     municipality 'г. Ульяновск'
     locality 'г. Ульяновск'
+    state { Member.state_machines[:state].states.map(&:name).first.to_s }
+    member_state { Member.state_machines[:member_state].states.map(&:name).first.to_s }
+    role 'user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.